Dockerfile

```diff
@@ -1,7 +1,14 @@
-FROM jinaai/jina:master
+FROM jinaai/jina:2.0.0rc8
 
-ADD *.py *.yml requirements.txt ./
+# install git
+RUN apt-get -y update && apt-get install -y git
 
+# install requirements before copying the workspace
+COPY requirements.txt /requirements.txt
```

Comment:
Why are we copying the requirements before the workspace?

Reply:
I copy this from some template. How about this? 
![image](https://user-images.githubusercontent.com/44033547/123292893-7cb68a80-d4e1-11eb-9697-b534f6718c16.png)